controllers/upgrade.go

```diff
@@ -108,6 +109,14 @@ func (r *RollingUpgradeReconciler) ReplaceNodeBatch(rollingUpgrade *v1alpha1.Rol
 				}
 			)
 
+			if nodeName == "" {
```

Comment:
Please remove this check so that I will investigate and address this issue accordingly. We shouldn't be coming to this point.